src/main/java/duke/TaskList.java

```diff
@@ -0,0 +1,115 @@
+package duke;
+
+import java.util.ArrayList;
+
+/**
+ * Encapsulates the TodoList which is the overall list of all Tasks in Duke.
+ */
+public class TaskList {
```

Comment:
Perhaps it is a better practice to not abuse static, which diminishes the purpose of OOP.